fix_bag_msg_def.py

```diff
@@ -116,6 +116,9 @@ def main():
             # don't have mapping, but are allowed to use local msg def: retrieve
             # TODO: properly deal with get_message_class failing
             sys_class = roslib.message.get_message_class(msg_type)
+            if sys_class is None:
+                print("Message class '" + msg_type + "' not found.")
+                exit(1)
```

Comment:
Could we throw a `ValueError` is something instead?

I like to have tracebacks when something fails.

Reply:
Done, and thanks for the lightning fast turnaround!